### ISSUE TITLE
[9.0 CE] Error while adding company_id field to ir.actions.report.xml by inheritance

### ISSUE THREAD

**Issue by guidev224** [odoo/odoo#27251]
Impacted versions: 9 CE (downloaded on 2018-09-24)

Steps to reproduce:
Use _inherit to add company_id field [`company_id = fields.Many2one('res.company')`] to `ir.actions.report.xml` model

Current behavior:
When I try to updated the module I get the following error:
`
File "C:\Program Files (x86)\Odoo 8.0-20171010\server\.\openerp\models.py", line 3312, in _read_from_database
  File "C:\Program Files (x86)\Odoo 8.0-20171010\server\.\openerp\sql_db.py", line 171, in wrapper
  File "C:\Program Files (x86)\Odoo 8.0-20171010\server\.\openerp\sql_db.py", line 247, in execute
ProgrammingError: column ir_act_report_xml.company_id does not exist
LINE 1: ...t_report_xml"."usage","ir_act_report_xml"."model","ir_act_re...
`

Expected behavior:
Add the `company_id `field to `ir.actions.report.xml`

Here the code I am using:

```
from openerp import api, fields, models

class IrActionsReportXml(models.Model):
    _inherit = 'ir.actions.report.xml'

    company_id = fields.Many2one('res.company' )
```

**Comment by sswapnesh** [comment]
This belongs to Custom module, not Odoo's Issue.
Ps: Restart server and upgrade module.

**Comment by Yenthe666** [comment]
Indeed, as @sswapnesh mentioned. Plus you've reported as 9.0 but I can clearly see it is an 8.0 (Odoo 8.0-20171010), which is end of life.

**Comment by guidev224** [comment]
@Yenthe666 you're right! Sorry for the misleading screenshot, when I first encountered the error I was in V9 then I tested it in v8 and got the error. But it is the same error so I think it is something worth checking in the V9 as it is still maintained. And I have found some people stuck with the same error. Check this [link in odoo](https://www.odoo.com/fr_FR/forum/aide-1/question/got-programmingerror-column-does-not-exist-131352) forum.
